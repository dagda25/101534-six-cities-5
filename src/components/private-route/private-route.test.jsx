import React from "react";
import renderer from "react-test-renderer";
import {PrivateRoute} from "./private-route";
import {Switch, Router} from "react-router-dom";
import browserHistory from "../../browser-history";


const authorizationStatus = `true`;
const path = `/`;
const exact = true;
const render = () => {
  return <div></div>;
};


describe(`Render PrivateRoute correctly`, () => {
  it(`Render PrivateRoute correctly`, () => {
    const tree = renderer
      .create(
          <Router history={browserHistory}>
            <Switch>
              <PrivateRoute authorizationStatus={authorizationStatus} path={path} exact={exact} render={render}/>
            </Switch>
          </Router>,
          {
            createNodeMock: () => {
              return {};
            }
          }
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

