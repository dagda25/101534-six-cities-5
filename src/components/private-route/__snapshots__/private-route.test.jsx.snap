// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PrivateRoute correctly Render PrivateRoute correctly 1`] = `null`;
